src/Command/CreateUserCommand.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Command;
+
+use App\Entity\User;
+use App\Repository\User\UserRepositoryInterface;
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\PasswordHasher\Hasher\UserPasswordHasherInterface;
+
+#[AsCommand(
+    name: 'app:create-user',
+    description: 'Creates a new user.',
+    aliases: ['app:add-user']
```

Comment:
```suggestion
    aliases: ['app:add-user'],
```

В реальной жизни мы не добавляем aliases, по крайней мере такие. Он не особо упращает работу с командой и даже наоборот привносит неправильное понятие, так как add и create имеют разный смысл